README.md

```diff
@@ -17,12 +17,20 @@ Software documentation is released under the Creative Commons license.
 * The custom [GeekDocs theme](https://geekdocs.de/) provides a bare-bones theme with templates and shortcodes for styling the website and content.
 * Specific .css customizations are added on top of GeekDocs in the /static/custom.css, /data/menu/, and /layouts/ files
 
-# Local Build Process
+# Installation and Download Process
 
 1. Download Hugo Extended v0.117.0 from [https://github.com/gohugoio/hugo/releases/](https://github.com/gohugoio/hugo/releases/tag/v0.117.0) and [install](https://gohugo.io/getting-started/installing/ "Install Hugo").
    See these instructions for manually installing the Hugo extended version in WSL: https://sal.as/post/install-hugo-on-wsl/
 
 2. Download this repository.
 
-3. Open the downloaded repository and run Hugo with `hugo serve`.
-   When complete, a local copy of the website is available by entering `localhost:1313` in a browser address bar.
+# Local Build Process
+
+1. Open the downloaded repository within your desired text editor and run Hugo with `hugo serve`. When complete, a local copy of the website is available by entering `//localhost:1313/` in a browser address bar.
+2. As changes within your chosen text editor or integrated development environment (IDE) are made, Hugo automatically rebuilds the local copy to include the intended modifications.
+
+# Making File Changes
+
+1. From your desired text editor (VS Code, Sublime Text, Atom, Notepad++, etc), utilize [CommonMark](https://spec.commonmark.org/0.31.2/) or [HTML](https://www.w3schools.com/html/default.asp) commands to input changes.
+2. Save any intended changes within the text editor and wait for modifications to appear on the local build.
+3. If using GitHub Desktop, ensure that all desired changes have been committed to your working branch prior to creating the final pull request.
```

Comment:
Step one is not a step, it is an instruction but not a step. So I suggest the following change:
```suggestion
1. Open your desired text editor (VS Code, Sublime Text, Atom, Notepad++, etc) and enter text changes using [CommonMark](https://spec.commonmark.org/0.31.2/) Markdown or [HTML](https://www.w3schools.com/html/default.asp) tags to format text.
2. Save your changes and wait for modifications to show in the local build.
3. Submit the pull request. 
   If using GitHub Desktop, make sure you commit all desired changes to your working branch before creating the final pull request.
```
Keep instructions simple, clear, concise, and of course correct. ;-)